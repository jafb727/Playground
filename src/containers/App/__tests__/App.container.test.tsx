/**
 * @author: Jose A Felix
 * @editor: Jose A Felix
 * @name: App.container.test
 * @description: App container test cases
 */

/* --------------------------------------------- */

/** @import Libraries */
import { act } from "@testing-library/react";

/** Components */
import Component from "./../../App";

/** Utilities */
import { basicSetup } from "./../../../test/setupTests";

/* --------------------------------------------- */

describe("App.container", () => {
   test("Passes shallow snapshot test", () => {
      act(() => {
         const component = basicSetup(Component);
         expect(component).toMatchSnapshot();
      });
   });
});
