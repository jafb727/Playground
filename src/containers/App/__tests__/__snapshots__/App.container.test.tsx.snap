// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.container Passes shallow snapshot test 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiBox-root css-0"
        data-testid="login"
        id="login"
      >
        <div
          class="fullPageBackground defaultBackground undefined MuiBox-root css-0"
          data-testid="background"
        >
          <div
            class="fullSizeCentered MuiBox-root css-0"
            id="workspace"
          >
            <header
              class="headerFullPage"
              id="header"
            >
              <div
                class="collectionContainer collection MuiBox-root css-1fn0841"
                data-testid="collection"
              >
                <div
                  class="MuiBox-root css-0"
                  data-testid="logo"
                >
                  <h4
                    class="MuiTypography-root MuiTypography-h4 logo css-95oyd8-MuiTypography-root"
                  >
                    MY
                  </h4>
                </div>
              </div>
            </header>
            <section
              class="fullSizeCentered"
              id="content"
            >
              <div
                class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation12 loginContentContainer css-1e1fzod-MuiPaper-root"
              >
                <div
                  class="collectionContainer fullSizeElement MuiBox-root css-1fn0841"
                  data-testid="collection"
                >
                  <div
                    class="fullSizeCentered MuiBox-root css-0"
                    data-testid="form-component"
                  >
                    <div
                      class="formHeader MuiBox-root css-0"
                    >
                      <div
                        class="MuiBox-root css-0"
                        data-testid="logo"
                      >
                        <h4
                          class="MuiTypography-root MuiTypography-h4 logo css-95oyd8-MuiTypography-root"
                        >
                          MY
                        </h4>
                      </div>
                    </div>
                  </div>
                  <div
                    class="  loginBackground fullSizeElement MuiBox-root css-0"
                    data-testid="background"
                  />
                </div>
              </div>
            </section>
            <footer
              class="footerFullPage"
              id="footer"
            >
              <div
                class="collectionContainer collection MuiBox-root css-1a9getn"
                data-testid="collection"
              >
                <div
                  class="MuiBox-root css-0"
                  data-testid="copyright-component"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body2 css-n6q3br-MuiTypography-root"
                  >
                    © 
                    All rights reserved.
                  </p>
                </div>
              </div>
            </footer>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiBox-root css-0"
      data-testid="login"
      id="login"
    >
      <div
        class="fullPageBackground defaultBackground undefined MuiBox-root css-0"
        data-testid="background"
      >
        <div
          class="fullSizeCentered MuiBox-root css-0"
          id="workspace"
        >
          <header
            class="headerFullPage"
            id="header"
          >
            <div
              class="collectionContainer collection MuiBox-root css-1fn0841"
              data-testid="collection"
            >
              <div
                class="MuiBox-root css-0"
                data-testid="logo"
              >
                <h4
                  class="MuiTypography-root MuiTypography-h4 logo css-95oyd8-MuiTypography-root"
                >
                  MY
                </h4>
              </div>
            </div>
          </header>
          <section
            class="fullSizeCentered"
            id="content"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation12 loginContentContainer css-1e1fzod-MuiPaper-root"
            >
              <div
                class="collectionContainer fullSizeElement MuiBox-root css-1fn0841"
                data-testid="collection"
              >
                <div
                  class="fullSizeCentered MuiBox-root css-0"
                  data-testid="form-component"
                >
                  <div
                    class="formHeader MuiBox-root css-0"
                  >
                    <div
                      class="MuiBox-root css-0"
                      data-testid="logo"
                    >
                      <h4
                        class="MuiTypography-root MuiTypography-h4 logo css-95oyd8-MuiTypography-root"
                      >
                        MY
                      </h4>
                    </div>
                  </div>
                </div>
                <div
                  class="  loginBackground fullSizeElement MuiBox-root css-0"
                  data-testid="background"
                />
              </div>
            </div>
          </section>
          <footer
            class="footerFullPage"
            id="footer"
          >
            <div
              class="collectionContainer collection MuiBox-root css-1a9getn"
              data-testid="collection"
            >
              <div
                class="MuiBox-root css-0"
                data-testid="copyright-component"
              >
                <p
                  class="MuiTypography-root MuiTypography-body2 css-n6q3br-MuiTypography-root"
                >
                  © 
                  All rights reserved.
                </p>
              </div>
            </div>
          </footer>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
